llama_stack/distribution/datatypes.py

```diff
@@ -117,6 +117,22 @@ class Provider(BaseModel):
     config: Dict[str, Any]
 
 
+class LogCategoryConfig(BaseModel):
+    category: str = Field(
+        default="server",
+        description="Logging category to apply this configuration to (all | core | server | router | inference | agents | safety | eval | tools | client)",
+    )
+    level: str = Field(default="info", description="Log level (debug | info | warning | error | critical)")
+
+
+class LoggerConfig(BaseModel):
+    category_levels: List[LogCategoryConfig] = Field(
```

Comment:
could this be a `dict`? `category => level`?

Reply:
yup, changed to that